README.md

```diff
@@ -3,9 +3,35 @@
 
 # OpenQA Public cloud Helper
 
-PublicCloud-Watcher or `pcw` is a web app which monitors, displays and deletes resources on various Cloud Service Providers (CSPs). It identifies abandoned instances by searching for certain tags and performs cleanup tasks. It also deletes old custom uploaded images. The behavior differs per CSP.
-
-The fastest way to run PublicCloud-Watcher is via the provided containers, as described in the [Running a container](#running-a-container) section.
+PublicCloud-Watcher (PCW) is a web app which monitors, displays and deletes resources on various Cloud Service Providers (CSPs).
+PCW has two main flows :
+
+1. Update run ( implemented in [ocw/lib/db.py](ocw/lib/db.py) )
+ Concentrates on deleting VMs (in case of Azure Resource Groups) . Each update run scans accounts defined in configuration file and
+and persists all results of each scan into sqlite database. If it is new entity than it obligatory gets TTL. TTL may be taken from tag `openqa_ttl` if
```

Comment:
duplicate and (see line before).

```suggestion
writes the obtained results into a local sqlite database. Newly discovered entities get assigned an obligatory time-to-life value (TTL). TTL may be taken from tag `openqa_ttl` if
```

Also is "obligatory" the right word or do you mean "default"? Obligatory means every entity gets a TTL value, but as we will see later, one can bypass with by setting `pcw_ignore`.

Reply:
suggestion accepted but I will section open due to your concerns with "obligatory"

Reply:
What do you mean with `I will section open due to your concerns with "obligatory"`?

Reply:
Correct me if I'm wrong but I don't see "obligatory" adding any value in this section. Obligatory is a strong word and here I don't see it highlighting anything meaningful, or did I miss something?

I would like to suggest to remove obligatory here, but leave the last word to you. IMHO it highlights something that is not there and I find it a bit off.

Reply:
I have no special favor exactly to word "obligatory" . My point here was to underline that every VM will get TTL so basically unless you put `pcw_ingore` it will be deleted after TTL defined time . but if you think it is clear without this word too I am fine to drop it.